infra/base-images/base-runner/test_all.py

```diff
@@ -80,6 +80,11 @@ def find_fuzz_targets(directory):
       continue
     if filename.startswith('jazzer_'):
       continue
+    if os.getenv('FUZZING_ENGINE') == 'centipede':
```

Comment:
Is there a centipede fuzzer binary that is seperate form the fuzz target? Maybe this should be excluded too, like afl-fuzz is on line 79.

Reply:
That binary will be excluded in [the next line](https://github.com/google/oss-fuzz/blob/d6ea2a1c4bd6eb6055ff30b438ff01ac37b67389/infra/base-images/base-runner/test_all.py#L84), becase it does not exist in the directory of the sanitized target binary:
```
${PROJECT_NAME}
|--unsanitized_target_binary
|--centipede_binary
|--${PROJECT_NAME}_${SANITIZER}
   |--sanitized_target_binary
|--other_dirs
```

Reply:
As discussed, this does not work here yet since this structure does not exist for the current cloud build setup. Let's just start with some simple filtering and exclude only the `centipede` binary. 

In our bad_build_check, we can:
1. If sanitized, only check if the binaries are instrumented
2. If unsanitized, only check if the fuzzers run.

Longer term, we can restructure the cloud build mounting such that this structure exists there too.

Reply:
Added [a commit](https://github.com/google/oss-fuzz/pull/8422/commits/b0c8477b8cdecbc9ebabb4386aea7f0bab395890) to keep the build test simple and ignore the complex file structure here.

Reply:
Do we still need to exclude the main `centipede` binary here?